conf/modules.config

```diff
@@ -238,6 +238,12 @@ process {
             ext.args2        = '-bS'
             ext.prefix       = {"${meta.id}.umi_unsorted"}
         }
+
+        withName: ".*DRAGMAP_ALIGN" {
+            ext.args         = { "--RGSM ${meta.read_group}" }
+            ext.args2        = '-bS'
+            ext.prefix       = {"${meta.id}.umi_unsorted"}
```

Comment:
this prefix should only be used when we have UMI, no?

Reply:
hm, so this one is within an if block that checks for umi (`if (params.umi_read_structure) { ...`), so that should already be the case, no?

Reply:
Well, the snappy error is persistent...I commented umi out again from the ci.yml to skip that test. If the conf/modules.config is correct in its current state, I suggest merging for now